ion/src/device/usb/calculator.cpp

```diff
@@ -9,6 +9,7 @@ namespace USB {
 namespace Device {
 
 void Calculator::PollAndReset(bool exitWithKeyboard) {
+  CM4.STCSR()->setTICKINT(false);
```

Comment:
Good catch. I think it'd make more sense to remove/reset the interrupt *before* calling `PollAndReset` though.

Reply:
Ok